setup.py

```diff
@@ -43,7 +43,7 @@ def get_docs_url():
         "ifaddr==0.1.7",
     ],
     extras_require={
-        "FSoE": ["fsoe_master==0.1.4+pr75b2"],
+        "FSoE": ["fsoe_master==0.1.4+g9115368a2"],
```

Comment:
This is going to be moved to pyproject.toml
I will update fsoe_master on this [PR-411](https://github.com/ingeniamc/ingeniamotion/pull/411)